components/Registration/index.tsx

```
@@ -416,9 +416,13 @@ const Registration: FunctionComponent<RegistrationProps> = ({
     try {
       // Retreiving Claim Signature from the backend signer service
       const claimDetails = await getClaimDetails();
-
+      const matadata = {
+        airdropId: activeWindow.airdrop_id.toString(),
+        airdropWindowId: activeWindow?.airdrop_window_id?.toString(),
+        address: 'dfjaslfjlda1231lkjfldsa12413',
```

why do we need this hardcoded value ?

---

It's hardcoded because address hash comes from backend. When Prashant send this address key in response of API will replace it. So till then Anand tell me to bind hardcoded value and raised PR to test if matadata transaction is working or not.